.github/workflows/release.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
       - name: Download dependencies
         run: |
-          sudo apt update && sudo apt install -y build-essential libpng-dev protobuf-compiler
+          sudo apt update && sudo apt install -y build-essential libpng-dev protobuf-compiler libwebp-dev
```

Comment:
Is this package still needed now that we use a go native webp library? (16b33770dfed5eebb2699d8b02bd53332ed686d8)